src/style.css

```
@@ -1 +1,57 @@
 /* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .stars {
+    display: flex;
+
+    .stars__star {
+      width: 16px;
+      height: 16px;
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+      background-image: url(images/star.svg);
```

```suggestion
      margin-right: 4px;
      background: url(images/star.svg) no-repeat center;
```

can be a bit simplified